unit-test/deleteTask.test.js

```diff
@@ -0,0 +1,64 @@
+import deleteTask from './deleteTask.js';
```

Comment:
- [ ] Kindly use the original function. You can use mock for functions that persists `localstorage`